### PR TITLE
Use default apps from cluster chart

### PR DESCRIPTION
**Pull request by nprokopic**
Towards https://github.com/giantswarm/roadmap/issues/3119

### What this PR does / why we need it

This PR adds required config templates and Helm values, so that cluster-aws uses default apps from cluster chart.

More details TBA. 

### Checklist

- [x] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

**Comment by nprokopic** [comment]
Testing on golem.

Successfully deployed a cluster with all default apps in cluster-aws and cluster chart:

### Management cluster resources

```
❯ clusterctl describe cluster -n org-nikola defapps003 --show-conditions all --grouping=false
NAME                                             READY  SEVERITY  REASON  SINCE  MESSAGE 
Cluster/defapps003                               True                     19m             
│           ├─ControlPlaneInitialized            True                     23m             
│           ├─ControlPlaneReady                  True                     19m             
│           └─InfrastructureReady                True                     25m             
├─ClusterInfrastructure - AWSCluster/defapps003  True                     25m             
│             ├─ClusterSecurityGroupsReady       True                     26m             
│             ├─InternetGatewayReady             True                     29m             
│             ├─LoadBalancerReady                True                     25m             
│             ├─NatGatewaysReady                 True                     26m             
│             ├─RouteTablesReady                 True                     26m             
│             ├─SubnetsReady                     True                     29m             
│             └─VpcReady                         True                     29m             
├─ControlPlane - KubeadmControlPlane/defapps003  True                     19m             
│ │           ├─Available                        True                     23m             
│ │           ├─CertificatesAvailable            True                     25m             
│ │           ├─ControlPlaneComponentsHealthy    True                     19m             
│ │           ├─EtcdClusterHealthy               True                     19m             
│ │           ├─MachinesCreated                  True                     20m             
│ │           ├─MachinesReady                    True                     19m             
│ │           └─Resized                          True                     19m             
│ ├─Machine/defapps003-5grt9                     True                     25m             
│ │             ├─APIServerPodHealthy            True                     21m             
│ │             ├─BootstrapReady                 True                     25m             
│ │             ├─ControllerManagerPodHealthy    True                     21m             
│ │             ├─EtcdMemberHealthy              True                     21m             
│ │             ├─EtcdPodHealthy                 True                     21m             
│ │             ├─HealthCheckSucceeded           True                     21m             
│ │             ├─InfrastructureReady            True                     25m             
│ │             ├─NodeHealthy                    True                     21m             
│ │             └─SchedulerPodHealthy            True                     21m             
│ ├─Machine/defapps003-lv4js                     True                     19m             
│ │             ├─APIServerPodHealthy            True                     19m             
│ │             ├─BootstrapReady                 True                     20m             
│ │             ├─ControllerManagerPodHealthy    True                     19m             
│ │             ├─EtcdMemberHealthy              True                     19m             
│ │             ├─EtcdPodHealthy                 True                     19m             
│ │             ├─HealthCheckSucceeded           True                     19m             
│ │             ├─InfrastructureReady            True                     19m             
│ │             ├─NodeHealthy                    True                     19m             
│ │             └─SchedulerPodHealthy            True                     19m             
│ └─Machine/defapps003-vnpgl                     True                     21m             
│               ├─APIServerPodHealthy            True                     20m             
│               ├─BootstrapReady                 True                     21m             
│               ├─ControllerManagerPodHealthy    True                     20m             
│               ├─EtcdMemberHealthy              True                     20m             
│               ├─EtcdPodHealthy                 True                     20m             
│               ├─HealthCheckSucceeded           True                     20m             
│               ├─InfrastructureReady            True                     21m             
│               ├─NodeHealthy                    True                     20m             
│               └─SchedulerPodHealthy            True                     20m             
└─Workers                                                                                 
  └─MachinePool/defapps003-nodepool0             True                     23m             
                ├─BootstrapReady                 True                     23m             
                ├─InfrastructureReady            True                     23m             
                └─ReplicasReady                  True                     29m
```

HelmReleases:

```
❯ kubectl get helmrelease -n org-nikola | grep defapps003
defapps003-aws-ebs-csi-driver            30m    True    Release reconciliation succeeded
defapps003-cilium                        30m    True    Release reconciliation succeeded
defapps003-cloud-provider-aws            30m    True    Release reconciliation succeeded
defapps003-coredns                       30m    True    Release reconciliation succeeded
defapps003-network-policies              30m    True    Release reconciliation succeeded
defapps003-vertical-pod-autoscaler-crd   30m    True    Release reconciliation succeeded
```

Apps:
```
❯ kubectl get apps -n org-nikola | grep defapps003
defapps003                               0.71.0-94461e3584793b6fd5b20c5b8eb734ee7dc243b9   31m          30m             deployed
defapps003-app-operator                  6.10.3                                            30m          30m             deployed
defapps003-aws-pod-identity-webhook      1.14.2                                            30m          18m             deployed
defapps003-capi-node-labeler             0.5.0                                             30m          22m             deployed
defapps003-cert-exporter                 2.9.0                                             30m          13m             deployed
defapps003-cert-manager                  3.7.4                                             30m          21m             deployed
defapps003-chart-operator                3.2.1                                             30m          22m             deployed
defapps003-chart-operator-extensions     1.1.2                                             30m          18m             deployed
defapps003-cilium-servicemonitors        0.1.2                                             30m          18m             deployed
defapps003-cluster-autoscaler            1.27.3-gs8                                        30m          13m             deployed
defapps003-etcd-k8s-res-count-exporter   1.10.0                                            30m          13m             deployed
defapps003-external-dns                  3.1.0                                             30m          20m             deployed
defapps003-grafana-agent                 0.4.3                                             21m          18m             deployed
defapps003-k8s-dns-node-cache            2.6.1                                             30m          12m             deployed
defapps003-kube-prometheus-stack         9.1.2                                             21m          21m             deployed
defapps003-kyverno                       0.17.9                                            16m          16m             deployed
defapps003-kyverno-policies              0.20.2                                            16m          13m             deployed
defapps003-kyverno-policy-operator       0.0.7                                             16m          13m             deployed
defapps003-metrics-server                2.4.2                                             30m          13m             deployed
defapps003-net-exporter                  1.18.2                                            30m          17m             deployed
defapps003-node-exporter                 1.19.0                                            30m          13m             deployed
defapps003-observability-bundle          1.3.4                                             30m          21m             deployed
defapps003-prometheus-agent              0.6.9                                             21m          21m             deployed
defapps003-prometheus-operator-crd       9.0.0                                             21m          21m             deployed
defapps003-promtail                      1.5.3                                             21m          21m             deployed
defapps003-security-bundle               1.6.3                                             30m          16m             deployed
defapps003-teleport-kube-agent           0.8.0                                             30m          22m             deployed
defapps003-vertical-pod-autoscaler       5.2.1                                             30m          18m             deployed
```

### Workload cluster resources

#### Nodes

```
❯ kubectl get nodes
NAME                                         STATUS   ROLES                  AGE   VERSION
ip-10-0-142-106.eu-west-2.compute.internal   Ready    control-plane,master   23m   v1.25.16
ip-10-0-146-85.eu-west-2.compute.internal    Ready    worker                 25m   v1.25.16
ip-10-0-241-237.eu-west-2.compute.internal   Ready    worker                 25m   v1.25.16
ip-10-0-248-112.eu-west-2.compute.internal   Ready    control-plane,master   21m   v1.25.16
ip-10-0-72-18.eu-west-2.compute.internal     Ready    control-plane,master   26m   v1.25.16
ip-10-0-97-110.eu-west-2.compute.internal    Ready    worker                 25m   v1.25.16
```

#### Pods

```
❯ kubectl get pods -A
NAMESPACE         NAME                                                                 READY   STATUS             RESTARTS      AGE
giantswarm        chart-operator-66655fd845-9jg4x                                      1/1     Running            0             24m
kube-system       aws-cloud-controller-manager-992r2                                   1/1     Running            1 (23m ago)   23m
kube-system       aws-cloud-controller-manager-dfn2r                                   1/1     Running            1 (25m ago)   25m
kube-system       aws-cloud-controller-manager-lpzg4                                   1/1     Running            1 (22m ago)   22m
kube-system       aws-pod-identity-webhook-app-5f569c87d9-42kvb                        1/1     Running            0             20m
kube-system       aws-pod-identity-webhook-app-5f569c87d9-9kk9z                        1/1     Running            0             20m
kube-system       aws-pod-identity-webhook-app-5f569c87d9-ggz8k                        1/1     Running            0             20m
kube-system       aws-pod-identity-webhook-restarter-28561825-hmhnw                    0/1     Completed          0             3m49s
kube-system       capi-node-labeler-4df6s                                              1/1     Running            0             24m
kube-system       capi-node-labeler-75zxq                                              1/1     Running            0             24m
kube-system       capi-node-labeler-cwjnj                                              1/1     Running            0             22m
kube-system       capi-node-labeler-gdr5q                                              1/1     Running            0             24m
kube-system       capi-node-labeler-gpxqt                                              1/1     Running            0             24m
kube-system       capi-node-labeler-slj6k                                              1/1     Running            0             24m
kube-system       cert-exporter-daemonset-476k2                                        1/1     Running            0             15m
kube-system       cert-exporter-daemonset-5kcgb                                        1/1     Running            0             15m
kube-system       cert-exporter-daemonset-7tmbb                                        1/1     Running            0             15m
kube-system       cert-exporter-daemonset-k9s7w                                        1/1     Running            0             15m
kube-system       cert-exporter-daemonset-w6gx5                                        1/1     Running            0             15m
kube-system       cert-exporter-daemonset-z9vw6                                        1/1     Running            0             15m
kube-system       cert-exporter-deployment-b7694799-g9g6w                              1/1     Running            0             15m
kube-system       cert-manager-app-9897c9fc4-p5hkr                                     1/1     Running            0             23m
kube-system       cert-manager-app-cainjector-75bbc96675-vcggv                         1/1     Running            0             23m
kube-system       cert-manager-app-webhook-6df9d57495-6zjx2                            1/1     Running            0             23m
kube-system       cert-manager-app-webhook-6df9d57495-qmgd6                            1/1     Running            0             23m
kube-system       cilium-9njx8                                                         1/1     Running            0             24m
kube-system       cilium-bvs8x                                                         1/1     Running            0             25m
kube-system       cilium-lf4vg                                                         1/1     Running            0             25m
kube-system       cilium-mx5xk                                                         1/1     Running            0             25m
kube-system       cilium-operator-86c9fb486c-vdwjl                                     1/1     Running            0             25m
kube-system       cilium-operator-86c9fb486c-ztxgs                                     1/1     Running            0             25m
kube-system       cilium-qgdb6                                                         1/1     Running            0             25m
kube-system       cilium-v4sn8                                                         1/1     Running            0             22m
kube-system       cluster-autoscaler-6f6844b668-xb2b5                                  1/1     Running            0             14m
kube-system       coredns-controlplane-84bd7c7655-dmqnd                                1/1     Running            0             24m
kube-system       coredns-workers-95fcb8cc9-9r27w                                      1/1     Running            0             24m
kube-system       coredns-workers-95fcb8cc9-vnmxq                                      1/1     Running            0             24m
kube-system       ebs-csi-controller-c4f9bcf5c-ldccx                                   5/5     Running            0             25m
kube-system       ebs-csi-node-d79gp                                                   3/3     Running            0             25m
kube-system       ebs-csi-node-nkp67                                                   3/3     Running            0             25m
kube-system       ebs-csi-node-z9hbh                                                   3/3     Running            0             25m
kube-system       etcd-ip-10-0-142-106.eu-west-2.compute.internal                      1/1     Running            0             23m
kube-system       etcd-ip-10-0-248-112.eu-west-2.compute.internal                      1/1     Running            0             22m
kube-system       etcd-ip-10-0-72-18.eu-west-2.compute.internal                        1/1     Running            0             27m
kube-system       etcd-k8s-res-count-exporter-94864df85-hc4cl                          1/1     Running            0             15m
kube-system       external-dns-6bbbffdf4d-r6jlg                                        1/1     Running            0             18m
kube-system       grafana-agent-75mkx                                                  2/2     Running            0             20m
kube-system       grafana-agent-jqb2f                                                  2/2     Running            0             20m
kube-system       grafana-agent-wj7vk                                                  2/2     Running            0             20m
kube-system       hubble-relay-5c995f7f5d-s8djw                                        1/1     Running            0             25m
kube-system       hubble-ui-56c45d649-2ddj9                                            2/2     Running            0             25m
kube-system       k8s-dns-node-cache-f5mn4                                             1/1     Running            0             14m
kube-system       k8s-dns-node-cache-gddd8                                             1/1     Running            0             14m
kube-system       k8s-dns-node-cache-hlj9b                                             1/1     Running            0             14m
kube-system       k8s-dns-node-cache-jb58f                                             1/1     Running            0             14m
kube-system       k8s-dns-node-cache-ps5tm                                             1/1     Running            0             14m
kube-system       k8s-dns-node-cache-zvd8z                                             1/1     Running            0             14m
kube-system       kube-apiserver-ip-10-0-142-106.eu-west-2.compute.internal            1/1     Running            0             23m
kube-system       kube-apiserver-ip-10-0-248-112.eu-west-2.compute.internal            1/1     Running            0             22m
kube-system       kube-apiserver-ip-10-0-72-18.eu-west-2.compute.internal              1/1     Running            0             27m
kube-system       kube-controller-manager-ip-10-0-142-106.eu-west-2.compute.internal   1/1     Running            0             24m
kube-system       kube-controller-manager-ip-10-0-248-112.eu-west-2.compute.internal   1/1     Running            0             22m
kube-system       kube-controller-manager-ip-10-0-72-18.eu-west-2.compute.internal     1/1     Running            0             27m
kube-system       kube-prometheus-stack-kube-state-metrics-67c7bbbc56-6vjrx            1/1     Running            0             23m
kube-system       kube-prometheus-stack-operator-794d87d68d-vl4fh                      1/1     Running            0             23m
kube-system       kube-scheduler-ip-10-0-142-106.eu-west-2.compute.internal            1/1     Running            0             23m
kube-system       kube-scheduler-ip-10-0-248-112.eu-west-2.compute.internal            1/1     Running            0             22m
kube-system       kube-scheduler-ip-10-0-72-18.eu-west-2.compute.internal              1/1     Running            0             27m
kube-system       metrics-server-676d54c976-kcvvm                                      1/1     Running            0             15m
kube-system       metrics-server-676d54c976-l6s5b                                      1/1     Running            0             15m
kube-system       net-exporter-8kd8s                                                   1/1     Running            0             19m
kube-system       net-exporter-fw7ww                                                   1/1     Running            0             19m
kube-system       net-exporter-g2jg5                                                   1/1     Running            0             19m
kube-system       net-exporter-lkrnx                                                   1/1     Running            0             19m
kube-system       net-exporter-p8mf5                                                   1/1     Running            0             19m
kube-system       net-exporter-tp6hq                                                   1/1     Running            0             19m
kube-system       node-exporter-node-exporter-6nctq                                    1/1     Running            0             14m
kube-system       node-exporter-node-exporter-8t9jd                                    1/1     Running            0             14m
kube-system       node-exporter-node-exporter-9h77g                                    1/1     Running            0             14m
kube-system       node-exporter-node-exporter-dvn22                                    1/1     Running            0             14m
kube-system       node-exporter-node-exporter-pw62b                                    1/1     Running            0             14m
kube-system       node-exporter-node-exporter-r87rv                                    1/1     Running            0             14m
kube-system       prometheus-prometheus-agent-0                                        2/2     Running            0             23m
kube-system       promtail-2ztjg                                                       1/1     Running            0             23m
kube-system       promtail-bv5hf                                                       1/1     Running            0             23m
kube-system       promtail-kfhwp                                                       1/1     Running            0             23m
kube-system       promtail-rvwbm                                                       1/1     Running            0             23m
kube-system       promtail-rz9fd                                                       1/1     Running            0             22m
kube-system       promtail-z57pb                                                       1/1     Running            0             23m
kube-system       teleport-kube-agent-0                                                1/1     Running            0             24m
kube-system       teleport-kube-agent-1                                                1/1     Running            0             23m
kube-system       vertical-pod-autoscaler-admission-controller-5bbfc844d9-j4p5l        1/1     Running            0             19m
kube-system       vertical-pod-autoscaler-admission-controller-5bbfc844d9-xb9qn        1/1     Running            0             19m
kube-system       vertical-pod-autoscaler-recommender-8fdd4c48f-88hb9                  1/1     Running            0             19m
kube-system       vertical-pod-autoscaler-updater-54b7fc465b-vgsvv                     0/1     CrashLoopBackOff   7 (40s ago)   19m
kyverno           kyverno-admission-controller-5d869b9889-6d679                        1/1     Running            0             18m
kyverno           kyverno-admission-controller-5d869b9889-dpxfr                        1/1     Running            0             18m
kyverno           kyverno-admission-controller-5d869b9889-fzwtf                        1/1     Running            0             18m
kyverno           kyverno-background-controller-64bc4d7bdd-gh7pz                       1/1     Running            0             18m
kyverno           kyverno-cleanup-admission-reports-28561820-r26g5                     0/1     Completed          0             8m49s
kyverno           kyverno-cleanup-cluster-admission-reports-28561820-twkxz             0/1     Completed          0             8m49s
kyverno           kyverno-kyverno-plugin-69c4844cf5-md6zl                              1/1     Running            0             18m
kyverno           kyverno-policy-reporter-5675cd4b55-r6l8p                             1/1     Running            0             18m
kyverno           kyverno-reports-controller-6dc9d9f964-lbsqb                          1/1     Running            0             18m
kyverno           kyverno-ui-749bcb576f-r469n                                          1/1     Running            0             18m
security-bundle   kyverno-policy-operator-65dcf8f8f-tzbtk                              1/1     Running            0             15m
```

:warning: vertical-pod-autoscaler-updater is crashlooping in the workload cluster. These are the logs:

```
kubectl logs -n kube-system vertical-pod-autoscaler-updater-54b7fc465b-vgsvv
I0421 14:27:07.681673       1 flags.go:57] FLAG: --add-dir-header="false"
I0421 14:27:07.681750       1 flags.go:57] FLAG: --address=":8943"
I0421 14:27:07.681755       1 flags.go:57] FLAG: --alsologtostderr="false"
I0421 14:27:07.681759       1 flags.go:57] FLAG: --evict-after-oom-threshold="10m0s"
I0421 14:27:07.681763       1 flags.go:57] FLAG: --eviction-rate-burst="1"
I0421 14:27:07.681767       1 flags.go:57] FLAG: --eviction-rate-limit="-1"
I0421 14:27:07.681776       1 flags.go:57] FLAG: --eviction-tolerance="0.5"
I0421 14:27:07.681780       1 flags.go:57] FLAG: --in-recommendation-bounds-eviction-lifetime-threshold="12h0m0s"
I0421 14:27:07.681785       1 flags.go:57] FLAG: --kube-api-burst="75"
I0421 14:27:07.681790       1 flags.go:57] FLAG: --kube-api-qps="50"
I0421 14:27:07.681795       1 flags.go:57] FLAG: --kubeconfig=""
I0421 14:27:07.681799       1 flags.go:57] FLAG: --log-backtrace-at=":0"
I0421 14:27:07.681806       1 flags.go:57] FLAG: --log-dir=""
I0421 14:27:07.681811       1 flags.go:57] FLAG: --log-file=""
I0421 14:27:07.681815       1 flags.go:57] FLAG: --log-file-max-size="1800"
I0421 14:27:07.681820       1 flags.go:57] FLAG: --logtostderr="true"
I0421 14:27:07.681824       1 flags.go:57] FLAG: --min-replicas="1"
I0421 14:27:07.681828       1 flags.go:57] FLAG: --one-output="false"
I0421 14:27:07.681833       1 flags.go:57] FLAG: --pod-update-threshold="0.1"
I0421 14:27:07.681838       1 flags.go:57] FLAG: --skip-headers="false"
I0421 14:27:07.681843       1 flags.go:57] FLAG: --skip-log-headers="false"
I0421 14:27:07.681847       1 flags.go:57] FLAG: --stderrthreshold="2"
I0421 14:27:07.681851       1 flags.go:57] FLAG: --updater-interval="1m0s"
I0421 14:27:07.681856       1 flags.go:57] FLAG: --use-admission-controller-status="true"
I0421 14:27:07.681860       1 flags.go:57] FLAG: --v="2"
I0421 14:27:07.681864       1 flags.go:57] FLAG: --vmodule=""
I0421 14:27:07.681869       1 flags.go:57] FLAG: --vpa-object-namespace=""
I0421 14:27:07.681884       1 main.go:82] Vertical Pod Autoscaler 1.1.0 Updater
I0421 14:27:07.784496       1 fetcher.go:99] Initial sync of StatefulSet completed
I0421 14:27:07.885312       1 fetcher.go:99] Initial sync of ReplicationController completed
I0421 14:27:07.986267       1 fetcher.go:99] Initial sync of Job completed
I0421 14:27:08.086699       1 fetcher.go:99] Initial sync of CronJob completed
I0421 14:27:08.187395       1 fetcher.go:99] Initial sync of DaemonSet completed
I0421 14:27:08.288294       1 fetcher.go:99] Initial sync of Deployment completed
I0421 14:27:08.388453       1 fetcher.go:99] Initial sync of ReplicaSet completed
I0421 14:27:08.388626       1 controller_fetcher.go:141] Initial sync of Deployment completed
I0421 14:27:08.388659       1 controller_fetcher.go:141] Initial sync of ReplicaSet completed
I0421 14:27:08.388668       1 controller_fetcher.go:141] Initial sync of StatefulSet completed
I0421 14:27:08.388674       1 controller_fetcher.go:141] Initial sync of ReplicationController completed
I0421 14:27:08.388680       1 controller_fetcher.go:141] Initial sync of Job completed
I0421 14:27:08.388688       1 controller_fetcher.go:141] Initial sync of CronJob completed
I0421 14:27:08.388702       1 controller_fetcher.go:141] Initial sync of DaemonSet completed
W0421 14:27:08.388710       1 shared_informer.go:459] The sharedIndexInformer has started, run more than once is not allowed
W0421 14:27:08.388733       1 shared_informer.go:459] The sharedIndexInformer has started, run more than once is not allowed
W0421 14:27:08.388735       1 shared_informer.go:459] The sharedIndexInformer has started, run more than once is not allowed
W0421 14:27:08.388744       1 shared_informer.go:459] The sharedIndexInformer has started, run more than once is not allowed
W0421 14:27:08.388753       1 shared_informer.go:459] The sharedIndexInformer has started, run more than once is not allowed
W0421 14:27:08.388782       1 shared_informer.go:459] The sharedIndexInformer has started, run more than once is not allowed
W0421 14:27:08.388792       1 shared_informer.go:459] The sharedIndexInformer has started, run more than once is not allowed
I0421 14:27:08.489063       1 updater.go:246] Rate limit disabled
I0421 14:27:08.992342       1 api.go:94] Initial VPA synced successfully
E0421 14:28:09.008242       1 api.go:153] fail to get pod controller: pod=kube-controller-manager-ip-10-0-248-112.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-248-112.eu-west-2.compute.internal, last error node is not a valid owner
E0421 14:28:09.008302       1 api.go:153] fail to get pod controller: pod=etcd-ip-10-0-248-112.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-248-112.eu-west-2.compute.internal, last error node is not a valid owner
E0421 14:28:09.009558       1 api.go:153] fail to get pod controller: pod=kube-scheduler-ip-10-0-72-18.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-72-18.eu-west-2.compute.internal, last error node is not a valid owner
E0421 14:28:09.010812       1 api.go:153] fail to get pod controller: pod=etcd-ip-10-0-142-106.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-142-106.eu-west-2.compute.internal, last error node is not a valid owner
E0421 14:28:09.010866       1 api.go:153] fail to get pod controller: pod=kube-controller-manager-ip-10-0-72-18.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-72-18.eu-west-2.compute.internal, last error node is not a valid owner
E0421 14:28:09.010925       1 api.go:153] fail to get pod controller: pod=kube-scheduler-ip-10-0-142-106.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-142-106.eu-west-2.compute.internal, last error node is not a valid owner
E0421 14:28:09.015304       1 api.go:153] fail to get pod controller: pod=kube-apiserver-ip-10-0-248-112.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-248-112.eu-west-2.compute.internal, last error node is not a valid owner
E0421 14:28:09.015346       1 api.go:153] fail to get pod controller: pod=kube-apiserver-ip-10-0-72-18.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-72-18.eu-west-2.compute.internal, last error node is not a valid owner
E0421 14:28:09.015421       1 api.go:153] fail to get pod controller: pod=kube-controller-manager-ip-10-0-142-106.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-142-106.eu-west-2.compute.internal, last error node is not a valid owner
E0421 14:28:09.015478       1 api.go:153] fail to get pod controller: pod=kube-apiserver-ip-10-0-142-106.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-142-106.eu-west-2.compute.internal, last error node is not a valid owner
E0421 14:28:09.015552       1 api.go:153] fail to get pod controller: pod=etcd-ip-10-0-72-18.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-72-18.eu-west-2.compute.internal, last error node is not a valid owner
E0421 14:28:09.015586       1 api.go:153] fail to get pod controller: pod=kube-scheduler-ip-10-0-248-112.eu-west-2.compute.internal err=Unhandled targetRef v1 / Node / ip-10-0-248-112.eu-west-2.compute.internal, last error node is not a valid owner
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x159129f]

goroutine 1 [running]:
k8s.io/autoscaler/vertical-pod-autoscaler/pkg/updater/priority.(*scalingDirectionPodEvictionAdmission).LoopInit(0xc00007e548, {0x1a1dda3?, 0xa?, 0x0?}, 0xc000b6c7e0)
	/gopath/src/k8s.io/autoscaler/vertical-pod-autoscaler/pkg/updater/priority/scaling_direction_pod_eviction_admission.go:111 +0x11f
k8s.io/autoscaler/vertical-pod-autoscaler/pkg/updater/logic.(*updater).RunOnce(0xc000332630, {0x1c97290, 0xc00024b8f0})
	/gopath/src/k8s.io/autoscaler/vertical-pod-autoscaler/pkg/updater/logic/updater.go:183 +0xb44
main.main()
	/gopath/src/k8s.io/autoscaler/vertical-pod-autoscaler/pkg/updater/main.go:127 +0x7ef
```

Not sure if this is because of the change in this PR.

**Comment by nprokopic** [comment]
Cluster that was created with apps in default-apps-aws successfully upgraded to a cluster with default apps in cluster-aws :tada:

3-step upgrade process (with simplified details for the sake of brevity):

1. **Upgrade default-apps-aws version**
  - Apps get paused (so that Charts on the WC are not deleted) :white_check_mark:
  - new Helm hooks are created (post-upgrade hook to propagate pause to bundled apps, post-delete hook to remove App CRs that stay behind due to finalizers, which happens because of above pause) :white_check_mark:
2. **Delete default-apps-aws**
  - All App CRs are deleted :white_check_mark:
  - Chart CRs stay on WC and apps are still running :white_check_mark:
3. **Upgrade cluster-aws to a new version with default apps**
  - cluster upgraded successfully :white_check_mark:
  - All new App CRs for default apps got created :white_check_mark:
  - Chart CRs (for default apps) in the WC got updated to reflect that they are now deployed by cluster-aws :white_check_mark:

I would merge these PRs now (after re-running cluster-test-suites, which I think I would have to do locally, due to nature of the breaking changes here):
- https://github.com/giantswarm/cluster/pull/127
- https://github.com/giantswarm/cluster-aws/pull/581
- https://github.com/giantswarm/default-apps-aws/pull/463

Other tools/components (I can remember now) that need changes are then (should not be a lot of work):
- capi-migration-cli
- kubectl-gs
- happa
- cluster-test-suites (upgrade suite will be failing until there is a new cluster-aws release)

P.S. I would probably merge the above PRs even without changes to mentioned tools/components, so we can upgrade clusters ASAP.

**Review by calvix — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
```
Comment (AndiDog) [inline CHANGELOG.md:9]: Don't we need a big fat "how to migrate" hint here, in order to take ownership of default apps? I would assume that as a cluster admin, I need to first switch to a certain `default-apps-aws` version (with the pause annotations), and only then bump to the upcoming cluster-aws-v0.76?

**Review by AndiDog — APPROVED** [review]

**Review by calvix — APPROVED** [review]

**Comment by nprokopic** [comment]
e2e tests from custom cluter-test-suites branch were successful! 🎉 (creation only, as upgrades have a custom step)

[e2e-cluster-aws-with-default-apps.log](https://github.com/giantswarm/cluster-aws/files/15309299/e2e-cluster-aws-with-default-apps.log)

image

Merging this in a moment!

**Pull request merged**